recipes/gtest/all/conanfile.py

```diff
@@ -97,12 +120,14 @@ def _configure_cmake(self):
 
     def build(self):
         for patch in self.conan_data["patches"][self.version]:
-            tools.patch(**patch)
+            tools.patch(patch)
         cmake = self._configure_cmake()
         cmake.build()
+        if self.options.gtest_build_tests or self.options.gmock_build_tests:
+            self.run(f"ctest")
 
     def package(self):
-        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy("LICENSE", dst="licenses", src=self.source_folder)
```

Comment:
```suggestion
        self.copy("LICENSE", dst="licenses", src=self._source_folder)
```

Reply:
With the above changes. the license of gtest is located in source_folder not in _source_folder.